src/main/java/com/agilethought/atsceapi/model/User.java

```diff
@@ -3,12 +3,15 @@
 import lombok.Data;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
+import javax.persistence.GeneratedValue;
```

Comment:
your project is broken, please fix it. 
![image](https://user-images.githubusercontent.com/73487119/109543846-6f861380-7a8c-11eb-9ac2-7869d3ea1b3f.png)